lib/Pool.js

```diff
@@ -274,12 +284,38 @@ Pool.prototype.stats = function () {
 Pool.prototype._ensureMinWorkers = function() {
   if (this.minWorkers) {
     for(var i = this.workers.length; i < this.minWorkers; i++) {
-      this.workers.push(new WorkerHandler(this.script));
+      this.workers.push(new WorkerHandler(this.script, {
+        forkArgs: this.forkArgs,
+        forkOpts: this.forkOpts,
+        debugPort: this.debugPortStart + i
+      }));
     }
   }
 };
 
 /**
+ * Ensure that the maxWorkers option is a positive integer
+ * @param {*} value
+ * @returns {boolean} returns true when value is a positive integer
+ */
+function validateMaxWorkers(maxWorkers) {
```

Comment:
Good idea to extract `validateMaxWorkers` and `validateMinWorkers` in a separate function, only you forget to actually use them :D